apps/condo/pages/employee/[id]/index.tsx

```diff
@@ -339,7 +339,9 @@ export const EmployeeInfoPage: PageComponentType = () => {
     const employeeWithSpecializations = { ...employee, specializations: organizationEmployeeSpecializations.map(scope => scope.specialization) }
 
     const updateEmployeeAction = OrganizationEmployee.useUpdate({}, () => refetch())
-    const softDeleteAction = OrganizationEmployee.useSoftDelete(() => refetch())
+    const softDeleteAction = OrganizationEmployee.useSoftDelete(() => {
+        if (window.location.pathname === `/employee/${employeeId}`) push('/employee/')
```

Comment:
You should double check that window is available, in server side rendering context you do not have window

Best to double check it, so 

if window && window.location ...